edgelet/iotedge/src/support_bundle.rs

```diff
@@ -257,6 +269,12 @@ where
         println!("Running docker inspect for {}", module_name);
         let mut inspect = ShellCommand::new("docker");
 
+        /***
+         * Note: this assumes using windows containers on a windows machine.
+         * This is the expected production scenario.
+         * Since the bundle command does not read the config.yaml, it will fail if Linux containers are used.
```

Comment:
`Since the bundle command does not read the config.yaml, it cannot use the `moby.runtime_uri` from there.`